app/models/asset.rb

```diff
@@ -280,5 +280,9 @@ def ensure_parent_document_url_is_valid
     unless uri && %w[http https].include?(uri.scheme)
       errors.add(:parent_document_url, "must be an http(s) URL")
     end
+
+    if uri && uri.host.start_with?("draft-origin") && !draft?
```

Comment:
Can I find the draft origin host using Plek? There didn't seem to be an obvious way to do it but I suspect it might be possible, and would be better than hard coding it here

Reply:
I honestly don't know. I can see that the govuk-config file uses plek in [a similar way](https://github.com/alphagov/asset-manager/blob/main/lib/govuk_configuration.rb#L15-L23) to what you're describing but not similar enough to actually use here

Reply:
I have had a play around on integration and I can't seem to persuade it to. There are a bunch of PLEK related env vars in the integration container but I'm not sure how they got there, given I can't see them in the helm charts or AWS secrets manager. Are you able to give this a thumbs up for now @JonathanHallam, unless you've got other objections?

Reply:
yep all good :)